docs-ref-conceptual/TOC.md

```diff
@@ -12,6 +12,7 @@
 ## [Output formats](format-output-azure-cli.md)
 ## [Query](query-azure-cli.md)
 ## [Subscriptions](manage-azure-subscriptions-azure-cli.md)
+## [Clouds](manage-cloud-sets-azure-cli.md)
```

Comment:
Does this need to be in a top-level node? I'd rather nest it under Subscriptions . The ToC for the CLI is already too much of a mess to begin with.

(note that this is more for our own organization , most readers will find this via search).